File: src/codes/Users_Manual/link/visualizer_calc.cpp
#include "autd3/link/visualizer.hpp"

auto autd = autd3::Controller::builder()
                .add_device(autd3::AUTD3(autd3::Vector3::Zero()))
                .open(autd3::link::Visualizer::builder());

autd3::Vector3 center = autd.geometry().center() + autd3::Vector3(0, 0, 150);
autd3::gain::Focus g(center);
autd.send(g);

std::vector points{center};
const auto p = autd.link().calc_field(points, autd.geometry(), Segment::S0, 0);
std::cout << "Acoustic pressure at (" << center.x() << ", " << center.y()
          << ", " << center.z() << ") = " << p[0] << std::endl;

File: src/codes/Users_Manual/link/visualizer_calc.cs
var autd = Controller.Builder().AddDevice(new AUTD3(Vector3d.zero)).Open(Visualizer.Builder());

var center = autd.Geometry.Center + new Vector3d(0, 0, 150);
var g = new Focus(center);
autd.Send(g);

var points = new List<Vector3d> { center };
var p = autd.Link.CalcField(points, autd.Geometry, Segment.S0, 0);
Console.WriteLine($"Acoustic pressure at ({center.x}, {center.y}, {center.z}) = ({p[0]})");